coordinator/src/node.rs

```diff
@@ -481,9 +481,14 @@ impl Node {
                                     "DLC Channel settle offer has been rejected. Setting position to back to open."
                                 );
 
-                        db::positions::Position::update_closing_position(
+                        db::positions::Position::update_position_state(
                             &mut connection,
                             node_id.to_string(),
+                            vec![
+                                // the closing price doesn't matter here.
+                                PositionState::Closing { closing_price: 0.0 },
```

Comment:
Can you explain this please: Shouldn't this only deal with positions which are currently in `Rollover`?

Reply:
We are missing a bit of context here to be exact. I guess a typed reject would help here.

Anyways this part is processed after the `Reject` has been executed.

E.g. if we are processing a reject of a `SettleOffer` the channel state will have reverted to `Established`, but the same is true if a `RolloverOffer` or a `RenewOffer` gets rejected.

Since we don't know we simply update any position of that user in `Closing` (SettleOffer) or `Rollover` (RolloverOffer)` back to `Open`

Reply:
Got it thank you for the explanation :)